swagger.yaml

```
@@ -778,16 +811,74 @@ paths:
           description: "Invalid group ID or user ID provided"
           schema:
             $ref: '#/definitions/ErrorResponse'
+        401:
+          $ref: '#/responses/UnauthorizedError'
         500:
           description: "Invalid request body or unexpected Cognito response"
           schema:
             $ref: '#/definitions/ErrorResponse'
+  /groups-report:
+    get:
+      tags:
+        - Groups
+      summary: "List groups and the membership of the groups"
+      description: "To return a list of :-
+          group description
+          group member user email
+          json by default 
+          or if Request Header Accept 'text/csv' then csv format, consisting of preview team description amd user email. 
+        Notable Consideration :- 
+          Included on report
+            groups acting as a role (i.e. publishers, admin)
+          Excluded from Report
+            groups that are empty of users"
```

This description was hard to understand and the formatting appears to be off. Consider enabling rich formatting for this field using `|` and potentially clarifying the wording:
```suggestion
      description: |
        The list of group memebership with the following fields:
    
        * group description
        * group member user email
    
        Format is JSON by default or csv format if Request Header Accept 'text/csv' is specified.
        
        Included on report: groups acting as a role (i.e. publishers, admin)
        Excluded from Report: groups that are empty of users"
```